pyproject.toml

```diff
@@ -63,12 +63,12 @@ name = "Miscellaneous internal changes"
 showcontent = false
 
 [tool.poetry.dependencies]
-python = ">=3.6,<3.9"
+python = ">=3.7,<3.9"
 coloredlogs = ">=10,<16"
-sanic = ">=19.12.2,<21.0.0"
-sanic-cors = "^0.10.0"
+sanic = "^21.6.0"
```

Comment:
oh actually this conflicts with what is currently defined for Rasa Open Source, do we also need to bump the priority here?

And should we allow for ">=21.6.0,<22.0.0" to simplify upgrades?

Reply:
^21.6.0 actually means exactly that if I am not mistaken. Minor + micro updates are allowed, but not major ones

Reply:
yes we should also upgrade it in rasa oss, I have created an issue for that: https://github.com/RasaHQ/rasa/issues/9315